styx-scheduler-service/src/test/java/com/spotify/styx/SystemTest.java

```diff
@@ -279,21 +289,23 @@ public void restoresActiveStatesFromBigtableRetriggered() throws Exception {
     givenWorkflowEnabledStateIs(HOURLY_WORKFLOW, true);
     givenNextNaturalTrigger(HOURLY_WORKFLOW, "2016-03-14T16:00:00Z");
 
-    givenStoredEvent(Event.triggerExecution(workflowInstance, "trig1"),                        0L);
-    givenStoredEvent(Event.created(workflowInstance, TEST_EXECUTION_ID_1, TEST_DOCKER_IMAGE),  1L);
-    givenStoredEvent(Event.started(workflowInstance),                                          2L);
-    givenStoredEvent(Event.terminate(workflowInstance, 30),                                    3L);
-    givenStoredEvent(Event.retry(workflowInstance),                                            4L);
-    givenStoredEvent(Event.started(workflowInstance),                                          5L);
-    givenStoredEvent(Event.terminate(workflowInstance, 30),                                    6L);
-    givenStoredEvent(Event.retryAfter(workflowInstance, 30000),                                7L);
-    givenStoredEvent(Event.halt(workflowInstance),                                             8L);
-    givenStoredEvent(Event.triggerExecution(workflowInstance, "trig2"),                        9L);
-    givenStoredEvent(Event.created(workflowInstance, TEST_EXECUTION_ID_1, TEST_DOCKER_IMAGE), 10L);
-    givenStoredEvent(Event.started(workflowInstance),                                         11L);
-    givenStoredEvent(Event.terminate(workflowInstance, 30),                                   12L);
-    givenStoredEvent(Event.retryAfter(workflowInstance, 30000),                               13L);
-    givenActiveStateAtSequenceCount(workflowInstance,                                         13L);
+    givenStoredEvent(Event.triggerExecution(workflowInstance, "trig1"),                       0L);
+    givenStoredEvent(Event.dequeue(workflowInstance),                                         1L);
```

Comment:
Same comment here as in the RunState test, this test should pass without modification.